File: Problema2/Empleado.cs
using System;

class Empleado
{
    private string nombre;
    private string apellidoPaterno;
    private double salarioMensual;

    public Empleado(string nombre, string apellidoPaterno, double salarioMensual)
    {
        this.nombre = nombre;
        this.apellidoPaterno = apellidoPaterno;

        if (salarioMensual > 0)
            this.salarioMensual = salarioMensual;
        else
            this.salarioMensual = 0.0;
    }

    public double ObtenerSalarioAnual()
    {
        return salarioMensual * 12;
    }

    public void AumentarSalario()
    {
        salarioMensual = salarioMensual * 1.10;
    }
}

class Program
{
    static void Main()
    {
        Empleado empleado1 = new Empleado("Juan", "Perez", 15000);
        Empleado empleado2 = new Empleado("Ana", "Lopez", 20000);

        Console.WriteLine("Salarios anuales originales:");
        Console.WriteLine("Empleado 1: $" + empleado1.ObtenerSalarioAnual());
        Console.WriteLine("Empleado 2: $" + empleado2.ObtenerSalarioAnual());

        empleado1.AumentarSalario();
        empleado2.AumentarSalario();

        Console.WriteLine("\nSalarios anuales con aumento del 10%:");
        Console.WriteLine("Empleado 1: $" + empleado1.ObtenerSalarioAnual());
        Console.WriteLine("Empleado 2: $" + empleado2.ObtenerSalarioAnual());
    }
}

File: Problema2/empleado.py
class Empleado:

    def __init__(self, nombre, apellido_paterno, salario_mensual):
        self.nombre = nombre
        self.apellido_paterno = apellido_paterno

        if salario_mensual > 0:
            self.salario_mensual = salario_mensual
        else:
            self.salario_mensual = 0.0

    def obtener_salario_anual(self):
        return self.salario_mensual * 12

    def aumentar_salario(self):
        self.salario_mensual = self.salario_mensual * 1.10


empleado1 = Empleado("Juan", "Perez", 15000)
empleado2 = Empleado("Ana", "Lopez", 20000)

print("Salarios anuales originales:")
print("Empleado 1: $", empleado1.obtener_salario_anual())
print("Empleado 2: $", empleado2.obtener_salario_anual())

empleado1.aumentar_salario()
empleado2.aumentar_salario()

print("\nSalarios anuales con aumento del 10%:")
print("Empleado 1: $", empleado1.obtener_salario_anual())
print("Empleado 2: $", empleado2.obtener_salario_anual())
